docs/packaging_policy.md

```diff
@@ -108,29 +108,51 @@ for each combination. There are some particular cases for this general rule:
 ## Options
 
 Recipes can list any number of options with any meaning, and defaults are up to the recipe itself. The CI cannot enforce anything
-in this direction. However, there are a couple of options that have a special meaning for the CI:
+in this direction. However, there are a couple of options that have a special meaning for the CI.
 
-* `shared` (with values `True` or `False`). The CI inspects the recipe looking for this option. If it is found, it will
+### Predeinfed Options and Known Defaults
+
+ConanCenter supports many combinations, these are outline in the [supported configurations](supported_platforms_and_configurations.md) document for each platform. 
+
+* `shared` (with values `True` or `False`). The CI inspects the recipe looking for this option. The **default should be `shared=False`** and will
    generate all the configurations with values `shared=True` and `shared=False`.
 
-   > Note.- The CI applies `shared=True` only to the package being built, while every other requirement will use their defaults
-   > (typically `shared=False`). It's important to keep this in mind when trying to consume shared packages from ConanCenter
+   > **Note**: The CI applies `shared=True` only to the package being built, while every other requirement will. It's important to keep this in mind when trying to consume shared packages from ConanCenter.
```

Comment:
⬆️  Check again

Reply:
I am sorry 😕 You will need to add a suggestion I am not sure I follow what it missing...

The default is bold in https://github.com/conan-io/conan-center-index/pull/12281/files#diff-14538b50bf191933a8611daa28b18ac68d53e3de39dea62d8c5f4a2f72d463b1R117

I used the same syntax for all three options

***

If it's not obvious enough my other idea was to make a table, which i'd love to add

Reply:
Here you are repeating the same line twice: _"It's important to keep this in mind when trying to consume shared packages from ConanCenter."_ 

The important information is that only the library being built will use explicit `shared=True`, the rest of their dependencies will use their default (typically `shared=False`). It means that everything will be linked in the `.dll` that is being build, so, if you consume package from ConanCenter using `*:shared=True` you won't be consuming the code/logic of all those shared libraries, but just from the lower ones in the dependency graph (all the others were linked inside those ones). This is very important, as overriding the version of the other (or changing some options,...) won't modify the behavior at all.

Reply:
that is very important.
here, there are several things you need to be aware of:
- only root package is built with `shared=True`, all its dependencies are still with `shared=False`
- you won't be able to override dependencies without re-build, as information is erased from package id by default
- also, it's important you may run into https://github.com/conan-io/conan/issues/9712.

Reply:
(you also may want to add a paragraph, like if it's not the desired effect, you may need to run conan with `--build=always` or `-o *:shared=True`)

Reply:
```suggestion
   > **Note**: The CI applies `shared=True` only to the package being built, while every other requirement will `shared=False`. To consume everything as a shared library you will set `--build=always` and/or `-o *:shared=True`)
```